ansible/group_vars/sim.yml

```diff
@@ -30,7 +30,8 @@ sim_installation_type: "repository" # "{{ install_from }}"
 
 # These fields are NOT suggested to be modifie
 sim_work_dir: /opt/sim-linux-amd64
-sim_config_dir: /etc/dolphin
+sim_config_dir: /etc/delfin
```

Comment:
Instead of hardcoding config_dir path, it should be extracted from delfin.conf file. Pleace create the config_dir inside your create_db.py script.

Reply:
done